Games/types/Mafia/items/IlluGun.js

```diff
@@ -0,0 +1,38 @@
+const Item = require("../Item");
+const Random = require("../../../../lib/Random");
+
+module.exports = class IlluGun extends Item {
+
+    constructor(reveal) {
```

Comment:
Mechanically I think the Illusionist gun should always reveal, else it invalidates the power of illu.